run-tests.ps1

```diff
@@ -1,3 +1,24 @@
-$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$ErrorActionPreference = 'Stop'
+$here = Split-Path -Parent $PSCommandPath
+$pesterVer = '4.10.1'
+try {
+    [array]$existingModule = Get-Module -ListAvailable Pester
+    if (!$existingModule -or ($existingModule.Version -notcontains $pesterVer)) {
+        Install-Module Pester -RequiredVersion $pesterVer -Force -Scope CurrentUser
+    }
+    Import-Module Pester
+    $results = Invoke-Pester $here/module/Corvus.Deployment.Module.Tests.ps1 -PassThru
```

Comment:
Is there any way to use a GLOB rather than a hard coded file name? Would that make it more reusable?